build.gradle

```diff
@@ -49,8 +49,6 @@ subprojects { project ->
   dependencies {
```

Comment:
update spinnaker-gradle-project to 3.8.0 and spring-boot-version to 1.2.8.RELEASE in this file